crates/matrix-sdk/src/room/member.rs

```diff
@@ -64,4 +64,22 @@ impl RoomMember {
         let request = MediaRequest { source: MediaSource::Plain(url.to_owned()), format };
         Ok(Some(self.client.media().get_media_content(&request, true).await?))
     }
+
+    /// Adds the room member to the current account data's ignore list
+    pub async fn add_to_ignore_list(&self) -> Result<()> {
```

Comment:
Why not just name this `ignore` (and the other one `unignore`)? Also I just realized you added two public ways to ignore the user, one through `RoomMember` and one through the `Account` / `Client`, both in the main crate as well as FFI. Does EX actually need both?

Reply:
i feel the client one will be useful when we will implement the ignore/unignore functionality through the list of ignored/blocked users from the settings (waaaay later), and also for ignoring when reporting content).
The Room Member one will be useful for when ignoring directly while browsing the room members of a room.

I changed the name slightly because someone pointed out it could be a bit confusing to call them `ignore` (since it could look like a local ignore of the room) so I wanted to be more precise, but honestly let me know what you think about because I feel the docs are already clear that this is putting the value inside the account data's ignore list

Reply:
I don't think the name is any clearer really. I think it should just be noted in documentation that this ignores the user across all rooms.

Reply:
okay then I'll revert to the previous name and add a more specific doc!

Reply:
Thanks!

Reply:
done